### ISSUE TITLE
Python 3.10 compatibility Issue with ` currentThread()` 

### ISSUE THREAD

**Issue by Vinalti** [rickysarraf/apt-offline#201]
When proceeding to a `apt-offline get` (on an update) the following error occurs.
```
DeprecationWarning: currentThread() is deprecated, use current_thread() instead
Exception in thread Thread-23 (run):############] 100.0% of 8 MiB
Traceback (most recent call last):
  File "/usr/lib/python3.10/threading.py", line 1016, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.10/threading.py", line 953, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3/dist-packages/apt_offline_core/AptOfflineLib.py", line 691, in run
    if threading.currentThread().guiTerminateSignal:
  File "/usr/lib/python3.10/threading.py", line 1449, in currentThread
    warnings.warn('currentThread() is deprecated, use current_thread() instead',
``` 

I fixed it locally. Would like to share the fix.

**Comment by Vinalti** [comment]
It appears that it is already fixed on this repo. Unfortunately the "apt install apt-offline" doesn't include the modifications yet.

**Comment by rickysarraf** [comment]
This should be available with the [1.8.5](https://github.com/rickysarraf/apt-offline/commit/9dff4ed97d8a7feb88714e59c4a477e7b98a11bb) release
